command/loadbot/loadbot_command.go

```diff
@@ -121,11 +192,35 @@ func (l *LoadbotCommand) Run(args []string) int {
 
 		return 1
 	}
+
+	convMode := Mode(strings.ToLower(mode))
+	if convMode != transfer && convMode != deploy {
+		l.Formatter.OutputError(errors.New("invalid loadbot mode"))
+
+		return 1
+	}
+
 	// maxConns is set to 2*tps if not specified by the user.
 	if maxConns == 0 {
 		maxConns = int(2 * tps)
 	}
 
+	// Parse the gas price
+	bigGasPrice, gasPriceErr := types.ParseUint256orHex(&gasPrice)
+	if gasPriceErr != nil {
+		l.Formatter.OutputError(fmt.Errorf("failed to decode gas price to value: %w", err))
+
+		return 1
+	}
+
+	// Parse the gas limit
+	bigGasLimit, gasLimitErr := types.ParseUint256orHex(&gasLimit)
+	if gasLimitErr != nil {
+		l.Formatter.OutputError(fmt.Errorf("failed to decode gas limit to value: %w", err))
+
+		return 1
+	}
```

Comment:
Failed to parse when --gas-price or --gas-limit is not given

Reply:
I was testing this behavior with the wrong parse func 🤦 

Fixed 👍 

6e01c9f